index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,700;1,700&family=Roboto:ital,wght@0,300;0,400;0,900;1,100&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburgerIcon"><img src="assets/Icon - Menu.png" /></div>
+    </header>
+    <main class="main">
+      <h1>
+        I’m Alexandra.<br />Glad to see<br />
+        you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h2 class="contactTitle">LET’S CONNECT</h2>
+      <ul>
+        <li><img src="assets/Vector.svg" alt="" /></li>
+        <li><img src="assets/Linkedin icon-2.svg" alt="" /></li>
+        <li><img src="assets/Vector-2.svg" alt="" /></li>
+        <li><img src="assets/Vector-3.svg" alt="" /></li>
+        <li><img src="assets/Vector-4.svg" alt="" /></li>
+      </ul>
+    </main>
```

Comment:
You have done very well in putting the main content of the page into the main tag 👍 However, I suggest that since this is only one section of the page, I suggest that you also put the headline part in a section tag. This is also semantically sound and boosts SEO scores 👍. Visit [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) to find out more